index.html

```diff
@@ -421,19 +383,91 @@
 
 
 
+            </div><!--MAIN ROW CLOSE-->
 
 
+            <!--***************** MAIN CONTENT *******************-->
 
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+            <script>
+                var username = 'eddyj9';
 
+                var baseUrl = 'https://api.github.com/users/' + username;
+                var repoUrl = baseUrl + '/repos';
+                var repoArea = $('#repo_area');
 
-        </div><!--MAIN ROW CLOSE-->
+                $.ajax({
+                    method: 'GET',
+                    url: baseUrl,
+                    dataType: 'json'
+                }).done(function (userInformation) {
+                    console.log("DONE");
 
+                    $('#user_email').replaceWith('eduardojimenez9@yahoo.com');
```

Comment:
`replaceWith` will remove the entire tag blowing away any formatting you have done.  `.text` is what you should be using.